src/error.ts

```diff
@@ -23,6 +23,7 @@ export class CogniteError extends Error {
       message += `\n${JSON.stringify(extra, null, 2)}`;
     }
     super(message);
+    Object.setPrototypeOf(this, CogniteError.prototype);
```

Comment:
?

Reply:
this is a bit nasty way js works
https://stackoverflow.com/questions/51229574/why-instanceof-returns-false-for-a-child-object-in-javascript

Reply:
I see. Maybe comment the link in the code?

Reply:
yep